### ISSUE TITLE
Lyon

### ISSUE THREAD

**Issue by dyoung123** [hudayousef/Week10#164]
Beautiful city with beautiful architecture

**Comment by AnjanGurz** [comment]
I agree! Lyon has amazing architecture and great food. It's definitely a beautiful city, and it's on my bucket list, too - would love to visit someday!
